includes/multi-currency/class-wc-payments-multi-currency-currency.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Class WC_Payments_Multi_Currency_Currency
+ *
+ * @package WooCommerce\Payments
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Multi Currency Currency object.
+ */
+class WC_Payments_Multi_Currency_Currency {
+
+	/**
+	 * Constructor.
+	 *
+	 * @param string $code Three letter currency code.
+	 * @param float  $rate The conversion rate.
+	 * @param bool   $auto Whether to enable by default or not, for dev purposes.
+	 */
+	public function __construct( $code = '', $rate = '', $auto = false ) {
+		$this->id   = strtolower( $code );
```

Comment:
I think that we should explicitly define the different properties of the `Currency` object at the top of the class e.g.

```php
/**
* Three letter currency code.
* @var string
*/
protected $code;
```

This will make it easier to access them via code completion in VSCode/PHPStorm.

An example of this pattern can be seen here: https://github.com/Automattic/woocommerce-payments/blob/develop/includes/class-wc-payments-action-scheduler-service.php

Reply:
Added and protected the props in https://github.com/Automattic/woocommerce-payments/pull/1795/commits/cdb969523e396de384ea946505bdcbb3a4b144b9

Reply:
Sorry to reopen this. But, shouldn't these properties be public rather than protected? Otherwise, we couldn't access them without any getter, right?

Reply:
Yes, I think in the case of the `Currency` class we probably would want them to be `public` so that we can easily access them outside of class context (since we're mainly just going to be wanting to get the info out of a currency for display on the frontend).

Apologies for confusing things with my example which used `protected`! 😄

Reply:
I noticed the protected properties were an issue after trying to move forward with the settings page, so I started creating a bunch of getters, but I ran out of time. It would be way easier to make them public instead.